doc/source/whatsnew/v0.17.0.txt

```diff
@@ -110,6 +110,7 @@ Other enhancements
 
 - ``pd.merge`` will now allow duplicate column names if they are not merged upon (:issue:`10639`).
 
+- ``msgpack`` submodule updated to 0.4.6 (:issue:`10581`)
 .. _whatsnew_0170.api:
```

Comment:
so if you use `pandas.msgpack` like this
```
>>> pandas.msgpack.unpackb(b'\x93\x01\x02\x03') # previous behavior
[1, 2, 3]
```
```
>>> pandas.msgpack.unpackb(b'\x93\x01\x02\x03') # new behavior
(1, 2, 3)
```

But if you use it this way
```
>>> pandas.io.packers.unpack(io.BytesIO(b'\x93\x01\x02\x03')).unpack()
(1, 2, 3)
```
then no change, because the `use_list` in `packers.unpack` is `False` by default already.